website/docs/r/env.html.markdown

```diff
@@ -0,0 +1,112 @@
+---
+subcategory: "core/v1"
+layout: "kubernetes"
+page_title: "Kubernetes: kubernetes_env"
+description: |-
+  The resource provides mechanisms to inject containers with environment variables 
+  that are managed specifically by Terraform.
+---
+
+# kubernetes_env
+
+The resource provides mechanisms to inject containers with environment variables 
+that are managed specifically by Terraform.
+Environment variables can be used to store fine-grained information such as port numbers and host names.
```

Comment:
I would say this line is not needed as they probably know what an environment variable is if they're looking at this page.